control/pzmap.py

```diff
@@ -75,16 +106,20 @@ def pzmap(sys, Plot=True, title='Pole Zero Map'):
 
     if (Plot):
         import matplotlib.pyplot as plt
+
+        if isdtime(sys):
```

Comment:
This needs to be `isdtime(sys, strict=True)`, otherwise it will be true for systems with unspecified timebases (which could be continuous or discrete).

Reply:
So we are treating systems with unspecified timebases as continuous (showing s-plane omega-damping - see next comment)?

Reply:
The idea is that if the timebase is unspecified, it can represent either a continuous-time or discrete-time system (since for many operations it doesn't matter).  Because of this, if you call `isdtime` on a system for a system in which `timebase == None`, `isdtime` will return true.  If you want something to only happen when a system is *really* a discrete time system, use `isdtime(sys, strict=True)`.